### DISCUSSION TITLE
Prime ng theme not working

### DISCUSSION THREAD

**Discussion by yashgithub2021** [primefaces/community#1323] [PrimeNG] [answered]
I have installed primeNg but the theme is only working when imported in style.css, not in angular.json. I dont want add in style.css

**Comment by nicosammito** [comment, marked as answer]
The styles array inside of the angular.json file is used to declare global styles for a project (styles.css being one).

If you import all of your other stylesheets into styles.css (and styles.css is specified in the styles array in angular.json), you'll end up with one global, composite stylesheet.

If you put all of your stylesheets into the angular.json styles array independently (rather than importing them into styles.css), you'll end up with multiple global stylesheets.

**Reply by yashgithub2021** [reply]
Ok got it, the issue is resolved thankyou :)
